locked-nft/contracts/NFTLocker.cdc

```diff
@@ -89,27 +93,177 @@ access(all) contract NFTLocker {
         return false
     }
 
-    /// The path to the NFTLocker Admin resource belonging to the Account
-    /// which the contract is deployed on
-    access(all) view fun GetAdminStoragePath(): StoragePath {
+    /// The path to the Admin resource belonging to the account where this contract is deployed
+    ///
+    access(all) view fun getAdminStoragePath(): StoragePath {
```

Comment:
i think this requires a change in the FE escrow transaction and we need to be careful when we upgrade the tx. We will need to sync it with the FE update

Reply:
i'll say we leave the naming fr now. Once FE moves to the new tx, then this can be renamed

Reply:
Good call, ty! Reverted the naming change 👍